cockroachcloud/managing-access.md

```diff
@@ -0,0 +1,407 @@
+
+#### Edit API key names
+
+To change the API key name for an existing API key:
+
+1. On the **Access Management** page, select the **Service Accounts** tab.
+1. Click the service account for which you want to create an API key to bring up the **Service Account Details** page.
+1. Find the API key ID in the **API Access** table.
+1. Click the **Action** button.
+1. Select **Edit**.
+1. In the **Edit API key name** dialog modify the API key name and click **Save changes**.
+
+## Manage SQL users on a cluster
+
+- [View all SQL users in your cluster](#view-all-sql-users-in-your-cluster)
+- [Change a SQL user's password](#change-a-sql-users-password)
+- [Remove a SQL user](#remove-a-sql-user)
+
+### Create a SQL user
+
+{{site.data.alerts.callout_danger}}
+By default, a new SQL user created using the UI or Cloud API is granted the SQL `admin` role. An `admin` SQL user has full privileges for all databases and tables in the cluster, and can create additional SQL users and manage their privileges.
+When possible, it is best practice to limit each user's privileges to the minimum necessary for their tasks, in keeping with the [Principle of Least Privilege (PoLP)](https://en.wikipedia.org/wiki/Principle_of_least_privilege).
+{{site.data.alerts.end}}
+
+<div class="filters clearfix">
+  <button class="filter-button page-level" data-scope="console">Use the {{ site.data.products.db }} Console</button>
+  <button class="filter-button page-level" data-scope="client">Use the SQL client</button>
+</div>
+<p></p>
+
+<section class="filter-content" markdown="1" data-scope="console">
+{% include cockroachcloud/cockroachcloud-ask-admin.md %}
+
+1. Navigate to your cluster's **SQL Users** page in the **Security** section of the left side navigation.
+1. Click the **Add User** button in the top right corner.
+
+    The **Create SQL user** modal displays.
+
+1. Enter a **Username**.
+1. Click **Generate & save password**.
+1. Copy the generated password and save it in a secure location.
+
+    Currently, all new users are created with full privileges. For more information and to change the default settings, see [Grant privileges to a SQL user](#grant-privileges-to-a-sql-user) and [Use SQL roles to manage access](#use-sql-roles-to-manage-access).
+</section>
+
+<section class="filter-content" markdown="1" data-scope="client">
+Once you have [connected to the cluster's SQL client](connect-to-your-cluster.html), you can create a new user.
+
+To create a new user, use the [`CREATE USER ... WITH PASSWORD`](../{{site.current_cloud_version}}/create-user.html) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> CREATE USER <username> WITH PASSWORD '<password>';
+~~~
+
+{{site.data.alerts.callout_info}}
+Be sure to create a password for each new user. Without a password, a user cannot connect to the cluster or access the DB Console. To add or change a password for a user, use the [`ALTER USER`](../{{site.current_cloud_version}}/alter-user.html) statement.
```

Comment:
Well, the password won't be needed if one is going to access the cluster using SSO via Cloud CLI, or if one uses client certificates which are themselves in limited access (still needs to be publicly documented).